<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsScreen</class>
 <widget class="QWidget" name="SettingsScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="background_img">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>600</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>900</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>900</width>
     <height>600</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../img/background.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="QPushButton" name="mainMenuButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: white;
	border: 2px solid black;
	border-radius: 10px;
}

QPushButton:hover {
	background-color: black;
	color: white;
}
</string>
   </property>
   <property name="text">
    <string>Return to Main Menu</string>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="redComboBox">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>150</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>145</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>Choose Color Sets</string>
   </property>
  </widget>
  <widget class="QComboBox" name="blueComboBox">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>185</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>215</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
  </widget>
  <widget class="QComboBox" name="greenComboBox">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>220</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Yellow</string>
   </property>
  </widget>

<widget class="QSlider" name="masterVolumeSlider">
  <property name="geometry">
   <rect>
    <x>160</x>
    <y>300</y>
    <width>200</width>
    <height>22</height>
   </rect>
  </property>
  <property name="orientation">
   <enum>Qt::Horizontal</enum>
  </property>
</widget>
<widget class="QLabel" name="labelMasterVolume">
  <property name="geometry">
   <rect>
    <x>60</x>
    <y>300</y>
    <width>100</width>
    <height>22</height>
   </rect>
  </property>
  <property name="text">
   <string>Master Volume</string>
  </property>
</widget>
<widget class="QSlider" name="soundEffectsSlider">
  <property name="geometry">
   <rect>
    <x>160</x>
    <y>350</y>
    <width>200</width>
    <height>22</height>
   </rect>
  </property>
  <property name="orientation">
   <enum>Qt::Horizontal</enum>
  </property>
</widget>
<widget class="QLabel" name="labelSoundEffects">
  <property name="geometry">
   <rect>
    <x>60</x>
    <y>350</y>
    <width>100</width>
    <height>22</height>
   </rect>
  </property>
  <property name="text">
   <string>Sound Effects</string>
  </property>
</widget>
<widget class="QSlider" name="musicVolumeSlider">
  <property name="geometry">
   <rect>
    <x>160</x>
    <y>390</y>
    <width>200</width>
    <height>22</height>
   </rect>
  </property>
  <property name="orientation">
   <enum>Qt::Horizontal</enum>
  </property>
</widget>
<widget class="QLabel" name="labelMusicVolume">
  <property name="geometry">
   <rect>
    <x>60</x>
    <y>390</y>
    <width>100</width>
    <height>22</height>
   </rect>
  </property>
  <property name="text">
   <string>Music</string>
  </property>
</widget>


  <widget class="QComboBox" name="yellowComboBox">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>255</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="applyChangesButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>480</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: white;
	border: 2px solid black;
	border-radius: 10px;
}

QPushButton:hover {
	background-color: black;
	color: white;
}
</string>
   </property>
   <property name="text">
    <string>Apply Changes</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
